date.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=@CMAKE_INSTALL_PREFIX@/bin
+libdir=@CMAKE_INSTALL_PREFIX@/lib@LIB_SUFFIX@
+includedir=@CMAKE_INSTALL_PREFIX@/include/date
+
+Name: date
+Description: A date and time library based on the C++11/14/17 <chrono> header
+Version: @PACKAGE_VERSION@
+Libs: -L${libdir} -ltz
```

Comment:
the `tz` library depends on the `BUILD_TZ_LIB` option. If that's off, then there are no libs to link.

Reply:
If I read this correctly, `BUILD_TZ_LIB` controls if this file even used to generate `date.pc` and later install.